policyengine_us/parameters/gov/states/ri/tax/income/deductions/standard/phase_out/increment.yaml

```diff
@@ -0,0 +1,15 @@
+description: Rhode Island phases the standard deduction out at this increment.
+values:
+  2021-01-01: 6_000
+  2022-01-01: 6_200
+  2023-01-01: 6_700
+
+metadata:
+  label: Rhode Island standard deduction phase out increment
+  period: year
+  unit: currency-USD
+  reference:
+  - title: Inflation-adjusted amounts set for tax year 2023
```

Comment:
add page number where exact number can be found - if multiple pages - all yaml and PR's